spec_test.go

```diff
@@ -8,67 +8,53 @@ import (
 	yaml "gopkg.in/yaml.v2"
 
 	"github.com/giantswarm/clustertpr"
-	"github.com/giantswarm/clustertpr/calico"
-	"github.com/giantswarm/clustertpr/cluster"
-	"github.com/giantswarm/clustertpr/customer"
-	"github.com/giantswarm/clustertpr/docker"
-	"github.com/giantswarm/clustertpr/docker/daemon"
-	"github.com/giantswarm/clustertpr/docker/registry"
-	"github.com/giantswarm/clustertpr/etcd"
-	"github.com/giantswarm/clustertpr/kubernetes"
-	"github.com/giantswarm/clustertpr/kubernetes/api"
-	"github.com/giantswarm/clustertpr/kubernetes/dns"
-	"github.com/giantswarm/clustertpr/kubernetes/hyperkube"
-	hyperkubedocker "github.com/giantswarm/clustertpr/kubernetes/hyperkube/docker"
-	"github.com/giantswarm/clustertpr/kubernetes/ingress"
-	"github.com/giantswarm/clustertpr/kubernetes/kubectl"
-	kubectldocker "github.com/giantswarm/clustertpr/kubernetes/kubectl/docker"
-	"github.com/giantswarm/clustertpr/kubernetes/kubelet"
-	"github.com/giantswarm/clustertpr/kubernetes/networksetup"
-	networksetupdocker "github.com/giantswarm/clustertpr/kubernetes/networksetup/docker"
-	"github.com/giantswarm/clustertpr/kubernetes/ssh"
-	"github.com/giantswarm/clustertpr/node"
-	"github.com/giantswarm/clustertpr/vault"
-	"github.com/giantswarm/kvmtpr/spec"
-	endpointupdater "github.com/giantswarm/kvmtpr/spec/endpointupdater"
-	k8skvm "github.com/giantswarm/kvmtpr/spec/k8skvm"
+	clustertprspec "github.com/giantswarm/clustertpr/spec"
+	clustertprdocker "github.com/giantswarm/clustertpr/spec/docker"
+	clustertprkubernetes "github.com/giantswarm/clustertpr/spec/kubernetes"
+	clustertprkuberneteshyperkube "github.com/giantswarm/clustertpr/spec/kubernetes/hyperkube"
+	clustertprkuberneteskubectl "github.com/giantswarm/clustertpr/spec/kubernetes/kubectl"
+	clustertprkubernetesnetworksetup "github.com/giantswarm/clustertpr/spec/kubernetes/networksetup"
```

Comment:
> Why should the alias of a package name rely on different rules than the package name itself?

Because: (1) it isn't visible outside a project and (2) unlike package it has to be unique inside the file.